controller_manager/controller_manager/spawner.py

```diff
@@ -175,22 +178,36 @@ def main(args=None):
             node.get_logger().info('Controller already loaded, skipping load_controller')
         else:
             if controller_type:
-                ret = subprocess.run(['ros2', 'param', 'set', controller_manager_name,
-                                      controller_name + '.type', controller_type])
-            ret = load_controller(
-                node, controller_manager_name, controller_name)
+                parameter = Parameter()
+                Parameter.name = prefixed_controller_name + '.type'
```

Comment:
Where is 'prefixed_controller_name' defined?

Reply:
There is now because we had to backport another PR.